### ISSUE TITLE
Design browser horizontal separaters should move with rows

### ISSUE THREAD

**Issue by luxaritas** [eternagame/EternaJS#387]
Right now the every-5-row separation lines are fixed relative to the screen. They should move with the rows as they scroll

**Comment by everyday847** [comment]
DesignBrowserMode implements this through the GridLines class, which has a fixed position during scrolling. A decent amount would have to change to implement this: there would have to be N/5 of them, rather than a screen height's worth, and they would have to respond to scroll events.

**Comment by everyday847** [comment]
I don't think this is an urgent need for v1.9: implementing this would be a departure from old DesignBrowser behavior. Boris said he wanted it and has a reasonable rationale for why it makes some sense to do it, but I find these lines helpful as they are -- now the vote is tied!

I have done some work on this and for some reason my math is just not making sense.

You'll want to put something in `setScrollVertical` like:

```
        const theme = UITheme.designBrowser;

        // shift positions
        const dataStart = theme.headerHeight + theme.filterHeight + theme.dataPadding;
        this._gridLines.position = new Point(
            10,
            dataStart + (this._firstVisSolutionIdx % 5) * theme.rowHeight
        );
```

That won't quite be right, though! You'll want something equivalent for the clickedSelectionBox.

**Comment by AlexHalb** [comment]
I can work on this
